app/src/main/res/values/dimens.xml

```diff
@@ -6,6 +6,20 @@
     <dimen name="splash_title_textsize">36sp</dimen>
     <dimen name="splash_subtitle_textsize">20sp</dimen>
 
+    <dimen name="edit_text_textsize">24sp</dimen>
+    <dimen name="edit_text_linespacingextra">3sp</dimen>
```

Comment:
3sp is extremely tiny, make sure you actually need this and cant just use a default value.